http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -239,6 +526,9 @@ paths:
             - openid
             - email
             - profile
+
+## Legacy Cluster API ##
+
   "/api/cluster/v2/{googleProject}/{clusterName}":
```

Comment:
shall we add `deprecated: true` to this route?

Reply:
Let's wait to do that until we're actually confident in the new routes - maybe after we switch automation tests over